console/console-init/ui/src/modules/address/dialogs/CreateAddress/Configuration.tsx

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
Can we qualify this as a container, as it's mainly calling graphQL for different scenarios?

Reply:
Yeah , but we discussed to put all modals like create and edit in dialogs folder